cmd/riscli/routers.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	pb "github.com/bio-routing/bio-rd/cmd/ris/api"
+	"github.com/bio-routing/bio-rd/util/log"
+	"github.com/urfave/cli"
+)
+
+// NewLPMCommand creates a new LPM command
+func NewGetRoutersCommand() cli.Command {
+	cmd := cli.Command{
+		Name:  "routers",
+		Usage: "get all routers and vrfs available",
+		Flags: []cli.Flag{},
+	}
+
+	cmd.Action = func(c *cli.Context) error {
+		conn := SetupGRPCClient(c)
+		defer conn.Close()
+
+		client := pb.NewRoutingInformationServiceClient(conn)
+		err := getRouters(client, c.GlobalString("router"), c.GlobalUint64("vrf_id"), c.GlobalString("vrf"))
+		if err != nil {
+			log.Errorf("Get Routers failed: %v", err)
+			os.Exit(1)
```

Comment:
Nit: Why not `log.Panicf()` instead of two error and exit? (Yes, I did that in the past...)